src/app/app.component.ts

```diff
@@ -58,6 +73,19 @@ export class AppComponent implements OnInit {
       .pipe(take(1))
       .subscribe((permission) => this.createMenu(permission));
 
+    this.subscriptions.push(
+      this.store
+        .select(selectFocusOrganizationName)
+        // eslint-disable-next-line ngrx/no-store-subscription
+        .subscribe((organizationName) => this.organizationControl.setValue(organizationName, { emitEvent: false }))
+    );
+
+    this.subscriptions.push(
+      this.organizationControl.valueChanges.subscribe((focusOrganizationName) =>
+        this.store.dispatch(setFocusOrganization({ focusOrganizationName }))
```

Comment:
What do you think about moving this line to the subscription lambda of the upper subscription?

Reply:
I agree with you, however, prettier is not happy with it. It automatically formats the code like this. And you can not change it :(